src/Uno.UWP/ApplicationModel/Background/BackgroundTaskRegistration.Android.cs

```diff
@@ -0,0 +1,114 @@
+﻿
+#if __ANDROID__
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.ApplicationModel.Background
+{
+
+	public partial class BackgroundTaskRegistration : IBackgroundTaskRegistration, IBackgroundTaskRegistration2, IBackgroundTaskRegistration3
+	{
+		public string Name { get; internal set; }
+		string IBackgroundTaskRegistration.Name => Name;
+		public IBackgroundTrigger Trigger { get; internal set; }
+
+		public Guid TaskId { get; internal set; }
+
+		private static void EnsureManifestPermission(string permission)
+		{
+			var context = Android.App.Application.Context;

```

Comment:
Use the PermissionHelper.

Reply:
Method is mentioned in comment in BackgroundTaskBuilder, not used in my code.
I'm changing comment, and removing method.